From f3e49453303ca52b6f32c988529aaa2253f514f4 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 5 Nov 2024 17:02:30 +0100
Subject: [PATCH 1344/1695] dts: st: fix length of SRAM1 regions

Fix length of SRAM1 region to fit the 64ko available.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c1914a226c8eae80833464ca759d5576c9a0b323 dts: st: fix length of SRAM1 regions]
Change-Id: I5a3bc819a5307f462e893bc380c560cea9865b59
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/416614
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-resmem.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-resmem.dtsi b/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-resmem.dtsi
index 466781236..80dd07aed 100644
--- a/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-resmem.dtsi
+++ b/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-resmem.dtsi
@@ -32,7 +32,7 @@
 		};
 
 		cm33_sram1: cm33-sram1@a061000 {
-			reg = <0x0 0xa061000 0x0 0x8000>;
+			reg = <0x0 0xa061000 0x0 0xf000>;
 			no-map;
 		};
 
-- 
2.47.2

